R/pandoc.R

```diff
@@ -624,6 +605,42 @@ is_highlightjs <- function(highlight) {
   !is.null(highlight) && (highlight %in% c("default", "textmate"))
 }
 
+resolve_highlight <- function(highlight, supported = highlighters()) {
+  is_supported <- TRUE
+  # for backward compatibility, partial match still need to work
+  highlight <- tryCatch(
+    error = function(e) { is_supported <<- FALSE; highlight },
+    match.arg(highlight, supported)
```

Comment:
Instead of `tryCatch(match.arg())`, I think you can just use `pmatch()` for partial matching.

Reply:
Indeed you're right. I simplified in fca7a75. 
Thanks!